package.json

```diff
@@ -123,7 +126,7 @@
     "l10n:check": "npm run l10n:extract && git diff-files --quiet --ignore-all-space locales && echo 'No missing strings'",
     "test:ci": "cross-env NODE_ENV=test nyc mocha --recursive --reporter mocha-junit-reporter --require frontend/test-setup.js frontend/test",
     "test:watch": "npm test -- --watch",
-    "test:all": "npm run lint && npm run flow && npm run test && npm run flow-coverage && cd addon && npm run lint && npm run coverage && npm run test",
+    "test:all": "npm run lint && npm run flow && npm run test && npm run flow-coverage && cd addon && npm run lint && npm run coverage && npm run test && tox -e ui-tests",
```

Comment:
Hmm, good question. I think you're probably right. I added both test:all and bootstrap. Maybe we could add the python / pip steps to bootstrap. I'd be fine with simplifying all the npm scripts in another way too. I'm not really sure I understand them all.